azure-pipelines.yml

```diff
@@ -296,35 +296,23 @@ jobs:
     displayName: 'Build C# Wrappers'
     inputs:
       solution: '$(Build.SourcesDirectory)\src\csharp\K4a.sln'
-      msbuildArgs: '/p:BaseOutputPath="$(Build.BinariesDirectory)\$(MSBuildConfiguration)\\"'
+      msbuildArgs: '/p:BaseOutputPath="$(Build.BinariesDirectory)\bin\$(MSBuildConfiguration)\\"'
```

Comment:
Why do we need the \\\\ after $(MSBuildConfiguration)?

Reply:
It was so that we don't escape the quote. But I did fix the csproj files so that we don't need the trailing slash. I could remove it.